test/torch/model/test_estimators.py

```diff
@@ -75,6 +76,13 @@
             num_batches_per_epoch=3,
             epochs=2,
         ),
+        lambda dataset: PatchTSTEstimator(
+            prediction_length=dataset.metadata.prediction_length,
+            patch_len=16,
+            batch_size=4,
+            num_batches_per_epoch=3,
+            trainer_kwargs=dict(max_epochs=2),
+        ),
```

Comment:
Would it be possible to add tests for `LagTST` and `DLinear` as well?

Reply:
yes sure let me add some tests for all these models